.gitignore

```diff
@@ -1,10 +1,28 @@
 .env
-src/docker/docker-compose.override.yml
+docker-compose.override.yml
```

Comment:
OK, but should it really be a global search pattern... do we expect docker-compose.override.yml file somewhere except docker dir?